### ISSUE TITLE
Extension stops working with "An unexpected error occurred" message

### ISSUE THREAD

**Issue by xolve** [filips123/PWAsForFirefox#98]
### Description

FirefoxPWA extension doesn't work and shows error message "An unexpected error occurred" on Setup screen, toolbar popup and debug console.

### Steps to Reproduce

With existing installation when I got the message to install runtime version 1.3.0 the Connector Installation page showed "Error: An unexpected error occurred" message.

The toolbar popup shows same message.

Upon removing and installing extension again same thing happens.

### Additional Information

The error happens with a new Firefox profile as well. I am using Nightly 97.0a1. Adding "DEBUG" file to %APPDATA%\FirefoxPWA doesn't produce detailed logs. `about:support` data is at https://gist.github.com/xolve/6e3b6cd6250809a1cc91e0325ee85fe0

### Environment

* Operating system: Windows 11
* System architecture: x86_64
* Installation method: MSI
* PWAsForFirefox extension version: 1.3.0
* PWAsForFirefox native version: 1.3.0
* Firefox version: 97.0a1

**Comment by mraxilus** [comment]
I get the same issue, when I try to run the connector from Powershell I get:

```
> firefoxpwa-connector.exe
Error: Failed to open registry key

Caused by:
    The system cannot find the file specified. (os error 2)
```

**Comment by Neeqstock** [comment]
Same issue here. PWAs not working anymore. Installed the update via Windows .msi installer

**Comment by mcl21014** [comment]
Same problem. Are there any logs we can share to help make it easier to resolve the issue?

**Comment by catsnsnacks** [comment]
Same here. No logs generated by adding DEBUG file for me either. Reverting to 1.2.2 allows existing PWAs to work again, but the extension forces a new tab with an upgrade message, so I cannot create new ones while staying on 1.2.2.

Operating system: Windows 11
System architecture: x86_64
Installation method: MSI
PWAsForFirefox extension version: 1.3.0
PWAsForFirefox native version: 1.3.0
Firefox version: 95.0

I will see if I can gather/output the errors from the F12 console and link them here if they are helpful

**Comment by filips123** [comment]
The problem seems to be that the MSI installer created a new registry key with new name while the program still expects the old one. I will soon release a fix.

**Comment by filips123** [comment]
This should be fixed in [v1.3.1](https://github.com/filips123/PWAsForFirefox/releases/tag/v1.3.1). Sorry for the inconvenience!

**Comment by xolve** [comment]
Thank you for the extension and maintaining it so diligently.
